app/controllers/api/v1/jobs_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::V1::JobsController < ApplicationController
+  skip_before_filter  :verify_authenticity_token
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.